cli/lsp/tsc.rs

```diff
@@ -2669,14 +2669,17 @@ pub enum SignatureHelpTriggerKind {
   Invoked,
   #[serde(rename = "retrigger")]
   Retrigger,
+  #[serde(rename = "unknown")]
+  Unknown,
```

Comment:
Please note this change, I had to add a "catch-all" in the match below, due to `SignatureHelpTriggerKind` being non-exhaustive enum

See https://github.com/gluon-lang/lsp-types/issues/213

Reply:
We need to check what impact it would have on tsc if we were to serialise that value, but we can do it as a followup.